components/semotus/test/syncStates/models/Address.ts

```diff
@@ -0,0 +1,63 @@
+import {property, Supertype, supertypeClass} from '../../../dist';
+import {Customer} from './Customer';
+import {Account} from './Account';
+import * as _ from 'underscore';
+
+@supertypeClass
```

Comment:
Do we have any test cases here where we're using the sync state property within the @supertypeClass decorator?